live-editing/configs/GridFinjsConfigGenerator.ts

```diff
@@ -11,6 +11,8 @@ import {
     IgxSwitchModule,
     IgxToggleModule
 } from "igniteui-angular";
+import { ContextDialogWithChartComponent
+} from "../../src/app/grid-finjs/context-dialog-with-chart/context-dialog-with-chart.component";
 import { FinJSDemoComponent } from "../../src/app/grid-finjs/grid-finjs-demo.component";
```

Comment:
you need to import AND use IgxCategoryChartModule in the config
import { IgxCategoryChartModule } from "igniteui-angular-charts/ES5/igx-category-chart-module";